recipes/coincurve/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "coincurve" %}
+{% set version = "18.0.0" %}
+{% set commit = "1dd0675add2da67ad9e6acfc47eed041066f2ea1" %}
+{% set sha256 = "124d212b1a663a02a8086d2094e58605d114698f6e562e1212e0ca27a326d84d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ofek/coincurve/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - update_pyproject.toml.patch
+    - update_setup.py.patch
+    - update_build.py.patch
+    - update_MANIFEST.in.patch
+    - update_coincurve_deprecated.patch
+
+build:
+  number: 0
+  script:  {{ PYTHON }} -m pip install . --use-pep517 -vvv
+  missing_dso_whitelist:
+    - "*/libc.so.6"
+    - "*/python3.dll"  # [win]
+    - "*/VCRUNTIME*.dll"  # [win]
+    - "*/ucrtbased.dll"  # [win]
+    - "*/KERNEL32.dll"  # [win]
+  ignore_run_exports:
+    - libsecp256k1
+    - cffi
+    - python
+    - vc40_runtime
+    - ucrt
+
+requirements:
+  host:
+    - {{ compiler('c') }}
+    - {{ pin_compatible('libsecp256k1') }}
+    - cffi >=1.3.0
+    - pip
+    - pkgconfig
+    - python
+  run:
+    - asn1crypto
+    - cffi >=1.3.0
+    - {{ pin_compatible('libsecp256k1') }}
+    - python
+
+test:
+  imports:
+    - {{ name }}
+  source_files:
+    - tests/
+  requires:
+    - {{ pin_compatible('libsecp256k1') }}
+    - pip
+    - pytest
+    - pytest-benchmark
+  commands:
+    - pip check
+    - pytest -v tests
+
+about:
+  home: https://github.com/ofek/coincurve
+  summary: Cross-platform Python CFFI bindings for libsecp256k1
+  doc_url: https://ofek.dev/coincurve/
+  description: |
+    This library provides well-tested Python bindings for libsecp256k1, 
+    the heavily optimized C library used by Bitcoin Core for operations 
+    on the elliptic curve secp256k1.
+  license: MIT
```

```suggestion
  license: MIT OR APACHE
```
Use the OR operator to list the identifiers for both licenses. Not sure if APACHE is the correct identifier name.

---

WIll do